kubernetes/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: machine-controller-manager
-        image:eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.28.0-dev-793b105c41adfc434e07107decaf1ac67fc1fd3f
+        image:eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.38.0
```

Comment:
I think there should be a space after the colon. `image: <image_url>`, isn't it?

Reply:
I will probably make this change myself 😄